extension/content_script.js

```diff
@@ -91,28 +91,48 @@ function move(type) { // Note: I have combined the "moveUp" and "moveDown" funct
 	} else {  // When the button is held down
 		if (!timer) {
 			(function repeat() { // Allows speed to be updated WHILE moving
-				timer = setTimeout(repeat, speed_adj);
+				console.log(speed_adj);
```

Comment:
Still need this for debugging?

Reply:
Sounds good! Will fix then re-pull request soon